app/controllers/entity_groups_controller.rb

```diff
@@ -0,0 +1,3 @@
+class EntityGroupsController < ApplicationController
+  before_action :authenticate_user!
+end
```

Comment:
@HFG43 [OPTIONAL] I kindly suggest you rename this controller to align it with the name of the **ExpenseGroup** model